core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -321,6 +315,11 @@ private static ColumnPage newLVBytesPage(TableSpec.ColumnSpec columnSpec,
   public abstract void setShortIntPage(byte[] shortIntData);
 
   /**
+   * Set boolean values to page
+   */
+  public abstract void setBooleanPage(byte[] booleanData);
```

Comment:
Since internally boolean is stored as byte, so you can implement this function to call setBytePage instead of abstract function

Reply:
ok, I have changed it

Reply:
ok, I have removed it